packages/engine/crawlers/solana/index.ts

```diff
@@ -19,6 +20,13 @@ export const solanaEngineRunner: SolanaRunner = {
 				currentPubkey,
 			);
 
+			try {
+				const { _id } = item;
+				const signatures = await getSignatureList(context.connection, _id);
+				await getTransactions(context, signatures, _id);
```

Comment:
with `await`, does this calling block the code below, can you use a promise and resolve them together